src/qt/transactionview.h

```diff
@@ -69,6 +70,12 @@ class TransactionView : public QWidget
     QLineEdit *addressWidget;
     QLineEdit *amountWidget;
 
+    QTimer *inputDelayPrefix;
```

Comment:
@lclc unfortunately there's mixed styles all over the place. Last time that was discussed, that I know of, is to use the defined convention in new or "touched" code.